### PR TITLE
WIP: frontend/feat time-slider

### PR DESCRIPTION
**Pull request by aherreDev**
Frontend changes needed:

Timeslider UI
- [ ] Worldmap to support date selection
- [x] Timeslider UI
- [x] Popup UI to support date selection
- [ ] Define default datapoints that assumes missing data from JSON as `no data`

### CODE REVIEW

**Comment by aherreDev** [comment]
I will push this branch in this days with the Time Slider UI component and Him DatePicker all with the full functionality (The connection with the services isn't include)

File: src/components/DateRangePicker.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+];
+
+class Dialog extends Component {
+  constructor() {
+    super();
+    this.state = {
+      days: [],
+      currentMonth: new Date().getMonth(),
+      lastSelection: false,
+      currentYear: 2020,
```
Comment (prahlad1989) [inline src/components/DateRangePicker.js:170]: new Date().getFullYear() will also work .

File: src/components/DateRangePicker.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+      enableArrows: false,
+    };
+    this.getMonthData = this.getMonthData.bind(this);
+    this.fillAllMonths = this.fillAllMonths.bind(this);
+    this.chooseDay = this.chooseDay.bind(this);
+    this.changeMonth = this.changeMonth.bind(this);
+  }
+  componentDidMount() {
+    const { currentMonth, allMonthsDate } = this.state;
+    let firstDay = new Date(2020, currentMonth, 1);
```
Comment (prahlad1989) [inline src/components/DateRangePicker.js:181]: is it better to avoid constants?

File: src/components/DateRangePicker.js (hunk 1 above)
Comment (aherreDev) [reply src/components/DateRangePicker.js:170]: For the currentYear state?

**Comment by aherreDev** [comment]
Timeslider UI
- [ ] Worldmap to support date selection
- [x ] Timeslider UI
- [ x] Popup UI to support date selection
- [ ] Define default datapoints that assumes missing data from JSON as `no data`

**Pull request merged**